kernel/main.c

```diff
@@ -340,6 +353,52 @@ async_init(void)
       panic("Unable to create a kthread for do_async_init()");
 }
 
+static void
+setup_uefi_runtime_services() {
+   EFI_RUNTIME_SERVICES *RT = TO_PTR(uefi_rt_addr);
```

Comment:
As mentioned in the email, there's no way to access this pointer that way. You'll have to use KERNEL_PA_TO_VA().